hack/aks/Makefile

```diff
@@ -116,13 +123,22 @@ endif
 	@$(MAKE) set-kubeconf
 
 overlay-byocni-nokubeproxy-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster without kube-proxy
+	$(AZCLI) network public-ip create --name "overlay-cni-no-kube-public-ip" \
+		--resource-group $(GROUP) \
+		--allocation-method Static \
+		--ip-tags "FirstPartyUsage=/TestTagging-O365" \
```

Comment:
Can you factorize this into a variable?
This gets replicated in many scenarios and we will have to change this IP at some point to use our own service tag and not this testing one.

Reply:
Done